odoo/modules/module.py

```diff
@@ -434,9 +434,13 @@ def get_test_modules(module):
         mod = importlib.import_module('.tests', modpath)
     except Exception as e:
         # If module has no `tests` sub-module, no problem.
-        if not pycompat.text_type(e).startswith(u'No module named'):
+        submodule_name = pycompat.text_type(e.name)
```

Comment:
mmm that may be slightly dangerous, I'm doubt *all* exceptions have a name attribute.

In fact it looks like ImportError doesn't even have a name attribute in P2...